packer/README.md

```diff
@@ -0,0 +1,89 @@
+# Cyber Hygiene Packer templates #
+
+## AMIs ##
+
+The following AMIs are available in this Packer template:
+
+| Template name | Description |
+| ------------- | ----------- |
+| bastion | Provides a jump box to a private VPC. |
+| dashboard | The Cyber Hygiene dashboard application. |
+| docker | Runs Docker configurations to perform BOD 18-01 and 20-01 scanning as well as generate the DHS [code.gov](https://code.gov) inventory. |
+| mongo | Provides the MongoDB database used by the Cyber Hygiene scanning system as well as running [cisagov/cyhy-commander]. |
+| nessus | A Nessus scanner for the Cyber Hygiene scanning system (referred to as a `vulnscanner`). |
+| nmap | An Nmap scanner for the Cyber Hygiene scanning system (referred to as a `portscanner`). |
+| reporter | Runs the daily notification and weekly report generation using [cisagov/cyhy-reports] |
+
+## Building ##
+
+Build an AMI with:
+
+```console
+cd packer
+ansible-galaxy install --role-file ansible/requirements.yml
+packer init .
+packer build -only amazon-ebs.<target AMI> .
+```
```

Comment:
We also should mention here how to build all of the AMIs at once like we do in the main repo README.